exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/ParquetRecordReader.java

```diff
@@ -73,6 +72,7 @@
   private static final long DEFAULT_BATCH_LENGTH = 256 * 1024 * NUMBER_OF_VECTORS; // 256kb
   private static final long DEFAULT_BATCH_LENGTH_IN_BITS = DEFAULT_BATCH_LENGTH * 8; // 256kb
   private static final char DEFAULT_RECORDS_TO_READ_IF_NOT_FIXED_WIDTH = 32*1024;
```

Comment:
while we are at it, I would probably rename DEFAULT_RECORDS_TO_READ_IF_NOT_FIXED_WIDTH to DEFAULT_RECORDS_TO_READ_VARIABLE_WIDTH